x-pack/plugins/actions/server/lib/task_runner_factory.ts

```diff
@@ -139,12 +139,18 @@ export class TaskRunnerFactory {
             ...getSource(references, source),
           });
         } catch (e) {
-          logger.error(`Action '${actionId}' failed: ${e.message}`);
+          const errorSource =
+            e instanceof ActionTypeDisabledError
+              ? TaskErrorSource.USER
+              : getErrorSource(e) || TaskErrorSource.FRAMEWORK;
+          logger.error(`Action '${actionId}' failed: ${e.message}`, {
```

Comment:
Would it make sense to do the same in the `catch` block inside `getActionTaskParams` below?

Reply:
That's a good idea @adcoelho! I went ahead and added it in this commit: https://github.com/elastic/kibana/pull/197818/commits/993d5e3b888747f48c91fab40aee970f37c0088a.